code/modules/mob/living/helper_procs.dm

```diff
@@ -42,3 +42,24 @@ default behaviour is:
 /mob/living/proc/isDeadorDying()	//returns 1 if dead or in crit
 	if(stat == DEAD || health <= config.health_threshold_crit)
 		return TRUE
+
+/mob/living/proc/make_zombie(mob/master)
+	to_chat(world, "make zombie called on [src]")
```

Comment:
delet this

Reply:
The rest of the code is already exploding. Give me a minute.